### PR TITLE
Fix randomizing blurbs bug, add a Clear Inputs button, and delete unused code.

### PR DESCRIPTION
**Pull request by NickLiggett**
# What does this PR do?
- [] New feature (adds functionality)
- [x] Debugging/Refactor

**Implementation/Changes:**

* Had to set the values of race and classs to a variable controlled by state in order to allow for the user to make their own choice AND select a random one.

* Also deleted Artificer and Blood Hunter from randomizerData because those are extra content and not in our API.

**Where should your reviewer start?**

* CharacterCreator component.

**What (if any) features are you implementing?**
*

**What (if anything) did you refactor?**
*

**What issues does this PR close?**
*

**Is there anything that you need from your teammate?**
*

**Any other comments, questions, or concerns?**
*

# How Has This Been Tested?
- [x] I tested my changes in the browser
- [] I tested with Cypress

# Checklist:
- [] My code follows the style guidelines of this project
- [] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [] My changes generate no new bugs
- [] I have added tests that prove my fix is effective or that my feature works
- [] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/Components/App.tsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ const App: FC = () => {
   return (
     <main>
       <Nav monsterViewHandler={() => monsterViewHandler()}/>
-      {(!monsterView) && <MainSection />}
+      {(!monsterView) && <CharacterSection />}
```
Comment (mayakappen) [inline src/Components/App.tsx:37]: I like the name character section for it more, good change!

File: src/Components/CharacterCreator.tsx [hunk 2]
```diff
@@ -11,13 +10,31 @@ type Props = {
 const CharacterCreator: FC<Props> = (props: Props) => {
 
     const [formData, setFormData] = useState<{name: string, race: string, classs: string, hp: number, ac: number, str: number, con: number, dex: number, wis: number, int: number, cha: number, about: string}>({name: '', race: '', classs: '', hp: 0, ac: 0, str: 0, con: 0, dex: 0, wis: 0, int: 0, cha: 0, about: ''})
-   
+    const [classsValue, setClasssValue] = useState<string | undefined>(undefined)
+    const [randomizing, setRandomizing] = useState<boolean>(false)
+    const [raceValue, setRaceValue] = useState<string | undefined>(undefined)
```
Comment (mayakappen) [inline src/Components/CharacterCreator.tsx:15]: Really elegant solution to our problem!

File: src/randomizerData.tsx [hunk 3]
```diff
@@ -49,7 +49,5 @@ export const classes: string[] = [
     'Rogue',
     'Sorcerer',
     'Warlock',
-    'Wizard',
-    'Artificer',
-    'Blood Hunter'
+    'Wizard'
```
Comment (mayakappen) [inline src/randomizerData.tsx:52]: I guess I can live with not being an artificer :(

File: src/Components/CharacterCreator.tsx [hunk 4]
```diff
@@ -31,24 +48,33 @@ const CharacterCreator: FC<Props> = (props: Props) => {
...
     }
-    // Need to move these methods to App.tsx ^^
 
+    const classValueHandler = (event: any) => {
+        props.setClass(event.target.value)
+        handleChange(event)
+    }
+    
+    useEffect(() => {
+        if (randomizing) {
```
Comment (mayakappen) [inline src/Components/CharacterCreator.tsx:61]: you are so smart

**Review by mayakappen — COMMENTED** [review]
Love the solution with the boolean that sets rando mode, really good job!

File: src/Components/App.tsx (hunk 1 above)
Comment (NickLiggett) [reply src/Components/App.tsx:37]: Yes it just makes more sense! Thanks!

File: src/randomizerData.tsx (hunk 3 above)
Comment (NickLiggett) [reply src/randomizerData.tsx:52]: I feel you... I'm a Blood Hunter in my campaign and love it! Maybe we can make a pay-wall for the users! Muahaha!

**Review by Ant-Shell — APPROVED** [review]
Awesome job, looks great!

**Pull request merged**